CHANGELOG.md

```diff
@@ -452,4 +460,5 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 [PR #978]: https://github.com/bareos/bareos/pull/978
 [PR #983]: https://github.com/bareos/bareos/pull/983
 [PR #991]: https://github.com/bareos/bareos/pull/991
+[PR #995]: https://github.com/bareos/bareos/pull/995
```

Comment:
These links are automatically updated before the release. It is not required to add them by hand. Also, PR 1000 should be mentioned here if really wanted to add the link.

Reply:
I've deleted them